app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -964,6 +975,12 @@ private VideoStream getSelectedVideoStream() {
         return sortedVideoStreams != null ? sortedVideoStreams.get(selectedVideoStreamIndex) : null;
     }
 
+    @Nullable
+    private AudioStream getSelectedAudioStream() {
```

Comment:
Rename this to a something like `getDefaultAudioStream()`, the current name is misleading, as this method does not really pick what the user selected. Maybe add a comment about the fact that it is going to be used in association with `getSelectedVideoStream()`.

Reply:
I pushed a change to this. Renamed and added a javadoc, let me know if it makes more sense.

(btw I also rebased to latest `dev`)